source/deployment/sso-saml-onelogin.rst

```diff
@@ -0,0 +1,127 @@
+Configure SAML with OneLogin
+=============================
+
+The following process provides steps to configure SAML with OneLogin for Mattermost.
+
+.. contents::
+  :backlinks: top
+  :local:
+
+.. include:: sso-saml-before-you-begin.rst
+
+Create a OneLogin connection app for Mattermost SSO
+---------------------------------------------------
+
+1. Add a SAML test connector app.
+  a. Sign into OneLogin as an administrator.
+  b. Go to **Apps > Add Apps**.
+  c. Search for "SAML Test Connector" and choose **SAML Test Connector (IdP) w/encrypt**.
+
+  .. image:: ../../source/images/onelogin_1_new_app.png
+
+  d. In the **Display Name** field, enter a name for the application, and optionally upload an app icon. You can use the Mattermost logo for the icon, which you can download from `Branding Guidelines <http://www.mattermost.org/brand-guidelines/>`_ page.
+
+  .. image:: ../../source/images/onelogin_2_basic_configuration.png
+
+  e. Make sure that the **Visible in portal** option is enabled.
+  f. Click **Save**.
+
+2. Configure the app.
+  a. Click the **Configuration** tab and enter the following values:
+    - **Recipient**: ``https://<your-mattermost-url>/login/sso/saml``
+    - **ACS (Consumer) URL**: ``https://<your-mattermost-url>/login/sso/saml``
+    - **ACS (Consumer) URL Validator**: ``https:\/\/<your-mattermost-url>\/login\/sso\/saml``
+
+  .. image:: ../../source/images/onelogin_3_configuration_1.png
+
+  b. Paste the Public Key that you generated earlier into the **SAML Encryption** field.
+
+  .. image:: ../../source/images/onelogin_4_configuration_2.png
+
+  c. Click **Save**.
+
+3. Enter the attribute parameters.
+
+  Attribute parameters map attributes between OneLogin and Mattermost. For more information on which attributes are configurable, see our :ref:`documentation on SAML configuration settings <saml-enterprise>`.
```

Comment:
again, looks like the relative link isn't working on GitHub preview (but may work on docs site?)